src/main/java/io/logz/logback/LogzioSender.java

```diff
@@ -129,9 +130,9 @@ private LogzioSender(String logzioToken, String logzioType, int drainTimeout, in
     }
 
     public static synchronized LogzioSender getOrCreateSenderByType(String logzioToken, String logzioType, int drainTimeout, int fsPercentThreshold, String bufferDir,
-                                                       String logzioUrl, int socketTimeout, int connectTimeout, boolean debug,
-                                                       LogzioLogbackAppender.StatusReporter reporter, ScheduledExecutorService tasksExecutor,
-                                                       boolean addHostname, String additionalFields, int gcPersistedQueueFilesIntervalSeconds) {
+                                                                    String logzioUrl, int socketTimeout, int connectTimeout, boolean debug,
```

Comment:
If only spaces added, let's revert to the original indentation please, so we can focus on what's been added